GliaWidgets/Sources/AlertManager/AlertManager.swift

```diff
@@ -0,0 +1,140 @@
+import UIKit
+import Foundation
+
+final class AlertManager {
+    private let environment: Environment
+    private let viewFactory: ViewFactory
+    private var composer: AlertTypeComposer
+    private var currentAlert: AlertInputType?
+    private var alertViewController: AlertViewController?
+    private var alertWindow: UIWindow?
+
+    init(
+        environment: Environment,
+        viewFactory: ViewFactory
+    ) {
+        self.environment = environment
+        self.viewFactory = viewFactory
+        composer = .init(
+            environment: .init(log: environment.log),
+            theme: viewFactory.theme
+        )
+    }
+}
+
+// MARK: - Public Methods
```

Comment:
That methods are not public.

Reply:
fixed [763c33b](https://github.com/salemove/ios-sdk-widgets/pull/968/commits/763c33b953fde15ccfb8400a27fb76f00b1e2646)